frontend/pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
can you revert this change when switching between the branches do `pnpm install --frozen-lockfile` to avoid this situation.

Reply:
I am taking care of it.